### ISSUE TITLE
Missing `stdlib.h` on old OSX systems

### ISSUE THREAD

**Issue by rdbisme** [otfried/ipe#240]
Hello, 
[I updated](https://github.com/macports/macports-ports/pull/4027) the `ipe` package provided by `macports`, and we're encountering a build failure on old OSX systems ([log](https://paste.macports.org/48aa4b94a1b3)), in particular:

```
Compiling ipelib.cpp...
/usr/bin/clang++ -arch x86_64 -mmacosx-version-min=10.10 -Wdeprecated-declarations -Wall -g -Os -fPIC -DIPEUI_COCOA -std=c++14 -I../include   -c -o ../../build/obj/ipelua/ipelib.o ipelib.cpp
Compiling ipeluageo.cpp...
/usr/bin/clang++ -arch x86_64 -mmacosx-version-min=10.10 -Wdeprecated-declarations -Wall -g -Os -fPIC -DIPEUI_COCOA -std=c++14 -I../include   -c -o ../../build/obj/ipelua/ipeluageo.o ipeluageo.cpp
Compiling ipeluaobj.cpp...
/usr/bin/clang++ -arch x86_64 -mmacosx-version-min=10.10 -Wdeprecated-declarations -Wall -g -Os -fPIC -DIPEUI_COCOA -std=c++14 -I../include   -c -o ../../build/obj/ipelua/ipeluaobj.o ipeluaobj.cpp
Compiling ipeluastyle.cpp...
/usr/bin/clang++ -arch x86_64 -mmacosx-version-min=10.10 -Wdeprecated-declarations -Wall -g -Os -fPIC -DIPEUI_COCOA -std=c++14 -I../include   -c -o ../../build/obj/ipelua/ipeluastyle.o ipeluastyle.cpp
Compiling ipeluapage.cpp...
/usr/bin/clang++ -arch x86_64 -mmacosx-version-min=10.10 -Wdeprecated-declarations -Wall -g -Os -fPIC -DIPEUI_COCOA -std=c++14 -I../include   -c -o ../../build/obj/ipelua/ipeluapage.o ipeluapage.cpp
Compiling ipeluaipelet.cpp...
/usr/bin/clang++ -arch x86_64 -mmacosx-version-min=10.10 -Wdeprecated-declarations -Wall -g -Os -fPIC -DIPEUI_COCOA -std=c++14 -I../include   -c -o ../../build/obj/ipelua/ipeluaipelet.o ipeluaipelet.cpp
ipeluaipelet.cpp:173:38: error: use of undeclared identifier 'getenv'
    ipeDebug("DYLD_LIBRARY_PATH=%s", getenv("DYLD_LIBRARY_PATH"));
                                     ^
ipeluaipelet.cpp:174:47: error: use of undeclared identifier 'getenv'
    ipeDebug("DYLD_FALLBACK_LIBRARY_PATH=%s", getenv("DYLD_FALLBACK_LIBRARY_PATH"));
                                              ^
2 errors generated.
make[1]: *** [../../build/obj/ipelua/ipeluaipelet.o] Error 1
make[1]: Leaving directory `/opt/local/var/macports/build/_Users_travis_build_macports_macports-ports_graphics_ipe/ipe/work/ipe-7.2.11/src/ipelua'
make: *** [ipelua] Error 2
```

So it seems somehow that `stdlib.h` is not included. It is something you can take care of upstream? (Maybe a conditional include on old OSX?)

**Comment by cjones051073** [comment]
Issue is simply a missing `#include <cstdlib>` include in `ipeluaipelet.cpp`. 
With a patch fixing this, the above MacPorts build is successful.
https://github.com/macports/macports-ports/commit/5a492077ab90915e28fdf7f0bf7e44cc87099e7d

**Comment by otfried** [comment]
Fixed in 7.2.12.
